torch/csrc/generic/methods/TensorMath.cwrap

```diff
@@ -1757,6 +1757,7 @@
   name: eye
   backends:
     - CPU
+    - CUDA
```

Comment:
I had a look at `THD` and it seems that it doesn't assert anything on CUDA implementation for `eye`. So should I just remove the `backends` flag, and let it work for distributed as well?